api/woeip/apps/core/views.py

```diff
@@ -2,7 +2,12 @@
 from django.http import JsonResponse
 from django.utils import timezone
 
+from rest_framework import generics, viewsets
+from rest_framework.decorators import action
+
 from .constants import Status
+from woeip.apps.core import models, serializers
+
 
 
 def health(_):
```

Comment:
Is this view needed anymore if it's gone from `urls.py`?

Reply:
Ah good catch, forgot to remove it here.